src/main/reactjs/public/i18n/fi-FI.json

```diff
@@ -175,6 +175,46 @@
         "register": "Rekisteri",
         "sendEmail": "Lähetä sähköposti"
       },
+      "clerkSendEmailPage": {
+        "title": "Lähetä sähköposti",
+        "sections": {
+          "recipients": "Vastaanottajat",
+          "subject": "Aihe",
+          "message": "Viesti"
```

Comment:
Aihe --> Aihe *
Viesti --> Viesti *

Reply:
Näitä on ehkä Figmassa päivitetty? Siellä on ihan vaan Aihe ja Viesti.

Reply:
joo, tosiaan riittää, että placeholderissa on tähti.